bin/check-es-cluster-health.rb

```diff
@@ -111,13 +111,13 @@ def run
     health = client.cluster.health options
     case health['status']
     when 'yellow'
-      if !options[:alert_status].include? 'red'
+      if !options[:alert_status].downcase.include? 'red'
```

Comment:
It probably would be better to just make the input limited, but I can change this after merge.